docs/source/pages/using.rst

```diff
@@ -205,3 +207,35 @@ Thanks to Datalog’s expressive language model, it’s easy to add exception ru
 requirement. For example, `the Mysql Connector/J <https://slsa.dev/spec/v0.1/requirements#build-service>`_ dependency in
 the Micronaut MuShop project does not pass the ``build_service`` check, but can be manually investigated and exempted if trusted. Overall, policies expressed in Datalog can be
 enforced by Macaron as part of your CI/CD pipeline to detect regressions or unexpected behavior.
+
+-----------------------------
+Analyzing a GitLab repository
+-----------------------------
+
+Macaron supports analyzing GitLab repositories, whether they are hosted on `gitlab.com <https://gitlab.com>`_ or on your self-hosted GitLab instance. The set up in these two cases are a little bit different.
+
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+Analyzing a repository on `gitlab.com <https://gitlab.com>`_
+''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+Analyzing a public repository on `gitlab.com <https://gitlab.com>`_ is quite similar to analyzing a public GitHub repository -- you just need to pass a proper GitLab repository URL to ``macaron analyze``.
+
+If you want to analyze some private repository hosted on ``gitlab.com``, you need to obtain a GitLab access token having at least the ``read_repository`` permission and store it into the ``MCN_GITLAB_TOKEN`` environment variable. For more detailed instructions, see `GitLab documentation <https://docs.gitlab.com/ee/user/profile/personal_access_tokens.html#create-a-personal-access-token>`_.
+
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''
+Analyzing a repository on a self-hosted GitLab instance
+'''''''''''''''''''''''''''''''''''''''''''''''''''''''
+
+To analyze a repository on a self-hosted GitLab instance, you need to do the following:
+
+- Add the following ``[git_service.gitlab.self_hosted]`` section into your ``.ini`` config. In :ref:`the default .ini configuration <action_dump_defaults>`, there is already this section commented out. You can start by un-commenting this section and modifying the ``domain`` value with the domain of your self-hosted GitLab instance.
```

Comment:
Instead of ```In :ref:`the default .ini configuration <action_dump_defaults>```, could we say something like
```
In the default .ini configuration (generated using :ref:`macaron dump-default <action_dump_defaults>`) ...
```
I think it's clearer for me, but I'm happy with using either ways.

Reply:
Fixed in ced46dc77a8983634292d2ed639d31e97c4fbd07.
I had to fix it this way since I'm not sure how to include inline code in a `:ref:` role.